components/raftstore/src/store/fsm/apply.rs

```diff
@@ -3445,7 +3445,7 @@ where
             .iter()
             .any(|res| res.region_id == self.delegate.region_id())
             && self.delegate.last_flush_applied_index != applied_index;
-        #[cfg(feature = "failpoint")]
+        #[cfg(feature = "failpoints")]
```

Comment:
How about removing it completely? It should be optimized away in release mode.

Reply:
Do you mean remove `apply_on_handle_snapshot_sync` or just the feature?

Reply:
Just the feature.